### ISSUE TITLE
Is it possible to use Pcap.Net to do IP Address Spoofing for websocket's in C#?

### ISSUE THREAD

**Issue by FirstBlood12** [PcapDotNet/Pcap.Net#121]
I am building a client in C# and i am testing how many connections my web server can handle.
Also i am using websocket-sharp library to connect to my web server,
but I want each connection to come from different(Spoofed) IP.
The only way I found to do this is using Pcap.Net but i don't know how can I modify source IP Address of websocket before is sent.
Can someone provide me example of doing it?

**Comment by KyeOnDiscord** [comment]
I don't think you can send requests to a web server with a "spoofed" ip, you'd have to use a vpn or proxies.

**Comment by FirstBlood12** [comment]
@kyeondiscord I know that once websocket is sent to the webserver with "spoofed" source IP Address that server will reply to the spoofed IP but my goal is how I can modify source IP Address of websocket using Pcap.Net?
